src/Grpc.AspNetCore.Server/GrpcEndpointRouteBuilderExtensions.cs

```diff
@@ -85,25 +85,33 @@ public static class GrpcEndpointRouteBuilderExtensions
                 throw new InvalidOperationException($"Error binding gRPC service '{typeof(TService).Name}'.", ex);
             }
 
+            var registery = builder.ServiceProvider.GetRequiredService<GrpcServiceDefinitionRegistry>();
+            registery.AddServiceDefinition(new GrpcServiceDefinition(typeof(TService), serviceBinder.ServiceMethods));
```

Comment:
PR also adds `GrpcServiceDefinitionRegistry`, a type in DI to get the services registered with gRPC.

Reply:
nit: typo "registery"